docker/preview/Jenkinsfile

```diff
@@ -122,7 +122,7 @@ def notifyBuild(String buildStatus = 'STARTED') {
   } else {
     color = 'RED'
     colorCode = '#FF0000'
-    summary = summary + " <!here>"
+    //summary = summary + " <!here>"
```

Comment:
le't leave it like that for now - I'll uncomment it when this pipeline will be the main one for preview release